.bazelrc

```diff
@@ -0,0 +1,67 @@
+# use a compiler name that doesn't symlink to ccache
+build --action_env=CC=clang-16 --host_action_env=CC=clang-16
+build --action_env=CXX=clang++-16 --host_action_env=CXX=clang++-16
+build --action_env=BAZEL_COMPILER=clang
+build --cxxopt=-std=c++20 --host_cxxopt=-std=c++20
+build --linkopt -fuse-ld=lld
+
+# https://github.com/bazelbuild/rules_foreign_cc/issues/1065
+# https://github.com/bazelbuild/rules_foreign_cc/issues/1186#issuecomment-2053550487
+build --host_action_env=CXXFLAGS=-Wno-int-conversion
+build --action_env=CXXFLAGS=-Wno-int-conversion
+build --host_action_env=CFLAGS=-Wno-int-conversion
+build --action_env=CFLAGS=-Wno-int-conversion
+
+# https://github.com/bazelbuild/bazel/issues/19915
+# https://github.com/bazelbuild/bazel/issues/20886
+build --noincompatible_sandbox_hermetic_tmp
```

Comment:
I think you can remove this if you bump to bazel 7.2

Reply:
yep, that seemed to work.

Reply:
I was thankful they got that into 7.2 https://github.com/bazelbuild/bazel/pull/22407